src/components/Calculator.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from 'react';
 import calculate from '../logic/calculate';
+import Quotes from './quotes';
```

Comment:
- [ ] The Quotes component should be imported into the main component, that is the App component, not the Calculator component. Kindly fix that.